code/randomized_algorithms/src/birthday_paradox/birthday_paradox.py

```diff
@@ -0,0 +1,12 @@
+# Part of Cosmos by OpenGenus Foundation
+
+import math
+
+# Function returns number of people that should be present for a given probability, p
+def compute_people(p):
+    
+    return math.ceil(math.sqrt(2 * 365 * math.log(1 / (1 - p))));
```

Comment:
[PEP 8 space rules](https://www.python.org/dev/peps/pep-0008/#other-recommendations)
`math.ceil(math.sqrt(2 * 365 * math.log(1 / (1 - p))))`
Should be:
`math.ceil(math.sqrt(2 * 365 * math.log(1 / (1-p))))`